patches/reconnecting-eventsource+1.5.0.patch

```diff
@@ -0,0 +1,108 @@
+diff --git a/node_modules/reconnecting-eventsource/build/esm/reconnecting-eventsource.js b/node_modules/reconnecting-eventsource/build/esm/reconnecting-eventsource.js
```

Comment:
Do we need this patch file?

Reply:
Ok. I'm not too fussed, as long as there'a reason for it being there :)

Reply:
There is, was mainly a comment to myself 😄